utils/python/CIME/build.py

```diff
@@ -309,18 +65,20 @@ def build_model(case, build_threaded, exeroot, clm_config_opts, incroot,
 
         # Make sure obj, lib dirs exist
         for build_dir in [objdir, libdir]:
-            if (not os.path.exists(build_dir)):
+            if not os.path.exists(build_dir):
                 os.makedirs(build_dir)
 
         file_build = os.path.join(exeroot, "%s.bldlog.%s" % (model, lid))
 
         # build the component library
-        stat = run_cmd("%s/buildlib %s %s %s >> %s 2>&1" % (config_dir, caseroot, bldroot, compspec, file_build),
-                       from_dir=os.path.join(exeroot, model), ok_to_fail=True)[0]
+        stat = run_cmd("%s/buildlib %s %s %s >> %s 2>&1" %
+                       (config_dir, caseroot, bldroot, compspec, file_build),
+                       from_dir=os.path.join(exeroot, model),
+                       ok_to_fail=True, verbose=True)[0]
         expect(stat == 0,
                "ERROR: %s.buildlib failed, see %s" % (comp, file_build))
 
-        for mod_file in glob.glob(os.path.join(objdir, "*_[Cc][Oo][Mm][Pp]_*")):
```

Comment:
This was copying object files which lead to a build error if you tried to clean and build again.